pytket/extensions/stim/backends/stim_backend.py

```diff
@@ -139,7 +139,7 @@ def __init__(self) -> None:
 
     @property
     def backend_info(self) -> Optional[BackendInfo]:
-        return self._backend_info
+        return None
```

Comment:
Should probably fully revert the change in the other PR?

Reply:
done